frontend/talentsearch/src/js/components/Router.tsx

```diff
@@ -92,556 +75,270 @@ const ReviewMyApplicationPage = React.lazy(
-      };
-    },
-  },
-  {
-    path: directIntakePaths.createApplication(":id"),
-    action: (context) => {
-      const poolId = context.params.id as string;
-      return {
-        component: <CreateApplication poolId={poolId} />,
-        authorizedRoles: [Role.Applicant],
-      };
-    },
-  },
-  {
-    path: directIntakePaths.signAndSubmit(":id"),
-    action: (context) => {
-      const applicationId = context.params.id as string;
-      return {
-        component: <SignAndSubmitPage id={applicationId} />,
-        authorizedRoles: [Role.Applicant],
-      };
-    },
-  },
-  {
-    path: directIntakePaths.applications(":userId"),
-    action: () => ({
-      component: <MyApplicationsPage />,
-      authorizedRoles: [Role.Applicant],
-    }),
-  },
-  {
-    path: directIntakePaths.reviewApplication(":poolCandidateId"),
-    action: (context) => {
-      const poolCandidateId = context.params.poolCandidateId as string;
-      return {
-        component: (
-          <ReviewMyApplicationPage poolCandidateId={poolCandidateId} />
-        ),
-        authorizedRoles: [Role.Applicant],
-      };
-    },
-  },
-];
-
-export const Router: React.FC = () => {
-  const intl = useIntl();
-  const locale = getLocale(intl);
-  const authPaths = useAuthRoutes();
-  const talentPaths = useTalentSearchRoutes();
-  const profilePaths = useApplicantProfileRoutes();
-  const directIntakePaths = useDirectIntakeRoutes();
-  const featureFlags = useFeatureFlags();
-  const { loggedIn } = React.useContext(AuthenticationContext);
-
-  const [result] = useGetAboutMeQuery();
-  const { data, fetching, error } = result;
-  const aiConnectionString = getRuntimeVariable(
-    "APPLICATIONINSIGHTS_CONNECTION_STRING",
-  );
-  if (aiConnectionString) {
-    const appInsights = new ApplicationInsights({
-      config: {
-        connectionString: aiConnectionString,
+const router = createBrowserRouter([
+  {
+    path: `/`,
+    element: <Layout />,
+    errorElement: <ErrorPage />,
+    children: [
+      {
+        path: ":locale",
+        errorElement: <ErrorPage />,
+        children: [
+          {
+            index: true,
+            element: <HomePage />,
+          },
+          {
+            path: "support",
+            element: <SupportPage />,
+          },
+          {
+            path: "search",
+            children: [
+              {
+                index: true,
+                element: <SearchPage />,
+              },
+              {
+                path: "request",
+                element: <RequestPage />,
+              },
+            ],
+          },
+          {
+            path: "register-info",
+            element: <RegisterPage />,
+          },
+          {
+            path: "logged-out",
+            element: <LoggedOutPage />,
+          },
+          {
+            path: "login-info",
+            element: <LoginPage />,
+          },
+          {
+            path: "create-account",
```

Comment:
Oops, the auto redirect here for new users doesn't seem to work anymore.

Reply:
Oops is right! 😓 

Fixed in [add create account redirect](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4597/commits/a866dffca30cc41415e009c560399536c27441fe)